src/src/LED.h

```diff
@@ -21,18 +21,9 @@ HslColor hslBlack(black);
 void updateLEDs(uint8_t isRXconnected, uint8_t tlm)
 {
 #if defined(PLATFORM_ESP32) && defined(GPIO_PIN_LED)
-    if (ExpressLRS_currAirRate_Modparams->enum_rate == RATE_200HZ)
-    {
-        strip.ClearTo(RgbColor(0, 0, LED_MAX_BRIGHTNESS));
-    }
-    if (ExpressLRS_currAirRate_Modparams->enum_rate == RATE_100HZ)
-    {
-        strip.ClearTo(RgbColor(0, LED_MAX_BRIGHTNESS, 0));
-    }
-    if (ExpressLRS_currAirRate_Modparams->enum_rate == RATE_50HZ)
-    {
-        strip.ClearTo(RgbColor(LED_MAX_BRIGHTNESS, 0, 0));
-    }
-  strip.Show();
+    #define ON(on) ((ExpressLRS_currAirRate_Modparams->enum_rate & on) ? 0 : LED_MAX_BRIGHTNESS)
+    strip.ClearTo(RgbColor(ON(0b100), ON(0b010), ON(0b001)));
+    strip.Show();
+    #undef ON
```

Comment:
I find this new version harder to read and understand (especially without comments) than the first one (though `else if` should be used). Is there a reason, that I am currently not aware of, for this rewrite, other than that it uses less LoC?